libs/domains/services/data-access/src/lib/domains-services-data-access.ts

```diff
@@ -395,6 +435,43 @@ export const mutations = {
     const response = await mutation(serviceId)
     return response.data
   },
+  async editService({ serviceId, payload }: ServiceRequest) {
```

Comment:
Here you don't need discrimated union because there is no difference between service type.
You can simplify it like this
```suggestion
  async editService({ serviceId, payload }: { serviceId: string, payload: AnyService }) {
```
and remove `ServiceRequest`